eos_data_distribution/producers/usb.py

```diff
@@ -107,7 +107,6 @@ def main():
     GLib.unix_signal_add(GLib.PRIORITY_DEFAULT, signal.SIGTERM, signal_cb)
     monitor = Gio.VolumeMonitor.get()
     store = SimpleStoreProducer(prefix=SUBSCRIPTIONS_SOMA,
-                                split=defaults.ENDLESS_NDN_CACHE_PATH,
```

Comment:
`self.split` was never used in `SimpleStore.Producer`. I think the mapping in `usb.py` was implemented by the fact that `publish_all_names()` is called with a different base path each time.